canonical-websites/www.ubuntu.com
# Moved /esm to the root

## Done

- Moved /support/esm to /esm
- Move the [copy doc](https://docs.google.com/document/d/1BK-XXECYJJllG8jIe3WN3X8EMo_3wBBa03nlzYDP8xE/edit#heading=h.4n9o6kows08v)

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/esm](http://0.0.0.0:8001/esm)
- Run through the following [QA steps](https://github.com/canonical-webteam/practices/blob/master/workflow/qa-steps.md)
- see that it still appears in the /support nav
- see that /support/esm redirects to /esm
- see that there are no links to /support/esm on the site

## Issue / Card

Fixes #4182
